docs/src/main/sphinx/release/release-362.md

```diff
@@ -0,0 +1,58 @@
+# Release 362 (XX Sep 2021)
+
+## General
+
+* Add support for {func}`LISTAGG`. ({issue}`4835`)
+* Improve join performance. ({issue}`8974`)
+* Improve performance of queries that contain `DISTINCT` clause. ({issue}`8967`)
+* Improve query performance by reducing CPU overhead of repartitioning data across worker nodes. ({issue}`9102`)
+* Prevent failure when querying `system.materialized_views` and materialized view is dropped concurrently. ({issue}`9050`) 
+* Fix `ALTER SCHEMA ... SET AUTHORIZATION` to resolve roles using the catalog of the schema instead of the session catalog. ({issue}`9066`)
+* Fix query failure when query contains a cast from `varchar` to a shorter `char`. ({issue}`9036`)
+* Fix planning of recursive queries when the recursion the base plan or the recursion step plan produce duplicate outputs.
+* Split metrics that are reported in operator stats into `connectorMetrics` and `metrics`. ({issue}`9156`)
+
+## Security
+
+* Add `IN catalog` clause to `CREATE ROLE`, `DROP ROLE`, `GRANT ROLE`, `REVOKE ROLE`, and `SET ROLE` to specify 
+  the target catalog of the statement instead of using the current session catalog. This change is necessary to 
+  support system roles in the future. The old behavior of these statements can be restored by setting the 
+  `deprecated.legacy-catalog-roles` config property. ({issue}`9087`)
+
+## BigQuery connector
+
+* Support case insensitive name matching for BigQuery views. ({issue}`9164`)
+* Change type mapping of BigQuery `datetime` from `timestamp(3)` to `timestamp(6)` in Trino. ({issue}`9052`)
```

Comment:
Are the docs for all these type mappings updated?

Reply:
Yes, they're already updated.
https://github.com/trinodb/trino/blob/master/docs/src/main/sphinx/connector/bigquery.rst#data-types